### PR TITLE
Update day 2 materials

### PR DESCRIPTION
**Pull request by nandan75**

### CODE REVIEW

**Comment by nandan75** [comment]
I have made changes between the tags

# CHANGES MADE -START

# CHANGES MADE -END

File: Scripts/Day2/notebooks/2.1_rstudio-downstream.md
Comment (georgiesamaha) [inline Scripts/Day2/notebooks/2.1_rstudio-downstream.md:1]: Removing this as we are recommending everyone use vs code. Is that ok @nandan75?

File: Scripts/Day2/notebooks/2.3_differentialex.md [hunk 1]
```diff
@@ -254,8 +255,34 @@ DESeq2::plotMA(res, ylim = c(-12, 12))
...
+
+A nice histogram of p-values will have a peak at the 0.05 end, and a uniform frequency at all other p-value bins. Think back to your null and alternate hypothesis. Under the null hypothesis, there is a 5% chance of genes will fall under p-value 0.05, 10 % for p-value under 0.1, etc. The high peak at the first bin (p-value 0 - 0.5) represents genes that reject the null hypothesis (in addition to all the false discoveries - hence our need to adjust for multiple testing!).
+
+A histogram of p-values that looks anything other than what is described above means that something weird has happened and you may need to contact your local statistician/bioinformatician. 
+
+This [blog post](http://varianceexplained.org/statistics/interpreting-pvalue-histogram/) has a nice explanation of each scenario if you want to explore this further.
+
+
+# CHANGES MADE -START
+  **Can you put this section as "Additional information? / plots"**
```
Comment (georgiesamaha) [inline Scripts/Day2/notebooks/2.3_differentialex.md:285]: Yep

File: Scripts/Day2/notebooks/2.3_differentialex.md [hunk 2]
```diff
@@ -238,7 +238,8 @@ Choose a significant gene that is downregulated in the knockout mice. Enter the
 </div>  
 
 
-# CHECK WHAT NEEDS TO BE RETAINED!
+# CHANGES MADE -START
```
Comment (georgiesamaha) [inline Scripts/Day2/notebooks/2.3_differentialex.md:241]: Done

File: Scripts/Day2/notebooks/2.4_enrichment.md [hunk 3]
```diff
@@ -15,23 +15,27 @@ fig-cap-location: top
...
 We now have a list of significant DE genes. To gain greater biological insights on the DE genes we can determine if there is enrichment of known biological functions, pathways or interactions.
 
+# CHANGES MADE -START
+
 There are different methods for identifying functional enrichments, e.g.:
 
-* Over representation analysis (ORA). These databases typically categorize genes into groups based on shared function, involvement in a biological pathway, presence in a specific cellular location, or other categorizations, e.g. functional pathways, etc.
-*  Functional class scoring (FCS) e.g. Gene set enrichment analysis (GSEA). 
-* Enrichment analysis can be done using some of the freely available web- and R-based tools. Today we will use a R-package "clusterprofile" to perform Over representation analysis. You can read more about the GSEA method at https://www.gsea-msigdb.org/gsea/index.jsp
+* **Over representation analysis (ORA)**: These methods typically categorize genes into groups based on shared function, involvement in a biological pathway, presence in a specific cellular location, or other categorizations, e.g. functional pathways, etc.
```
Comment (georgiesamaha) [inline Scripts/Day2/notebooks/2.4_enrichment.md:22]: done

File: Scripts/Day2/notebooks/2.4_enrichment.md [hunk 4]
```diff
@@ -15,23 +15,27 @@ fig-cap-location: top
...
+* **Functional class scoring (FCS)** e.g. Gene set enrichment analysis (GSEA). 
+* Enrichment analysis can be done using some of the freely available web- and R-based tools. Today we will use a R-package "clusterprofiler" to perform Over representation analysis. You can read more about the GSEA method at https://www.gsea-msigdb.org/gsea/index.jsp
+
+# CHANGES MADE -END
 
-In this workshop, we will learn how to perform gene ontology enrichment analysis.
+In this workshop, we will learn how to perform "Over representation analysis (ORA)" using gene ontology databases.
```
Comment (georgiesamaha) [inline Scripts/Day2/notebooks/2.4_enrichment.md:28]: changed

**Review by georgiesamaha — APPROVED** [review]
Changes made, pushed to main on [quarto materials](https://sydney-informatics-hub.github.io/rnaseq-workshop-2023/)

**Pull request merged**